### DISCUSSION TITLE
How can I develop a new integration?

### DISCUSSION THREAD

**Discussion by bfontaine** [chatwoot/chatwoot#10248] [Q&A] [answered]
Hello,
We’re considering migrating to Chatwoot but we need an integration with Magento so that agents can quickly get the last order(s) of the customer. How can I develop the integration? I see nothing in the docs.

Related: https://github.com/orgs/chatwoot/discussions/6857 (no response)

**Comment by sojan-official** [comment, marked as answer]
The easiest option to handle custom use cases is to use dashboard apps : https://www.chatwoot.com/hc/user-guide/articles/1677691702-how-to-use-dashboard-apps 

We will be happy to work with you if you want to contribute an integration to the core, but at the moment documentation is little sparse, you might have to refer one of the previous integration PRs to take it further forward.

**Reply by bfontaine** [reply]
Thank you!
